### PR TITLE
add ability to pass a custom logging function

### PR DESCRIPTION
**Pull request by sindresorhus**
I need it for my Alfred plugins where I don't want to use `console.error`.

// @jamestalmage

### CODE REVIEW

**Comment by jamestalmage** [comment]
LGTM

**Comment by jamestalmage** [comment]
maybe add a test?

File: .travis.yml [hunk 1]
```diff
@@ -2,5 +2,4 @@ language: node_js
 node_js:
   - '6'
   - '4'
-  - '0.12'
```
Comment (sindresorhus) [inline .travis.yml]: I don't have the time to debug dumb 0.12 issues.

**Comment by sindresorhus** [comment]
Done

**Pull request merged**